### ISSUE TITLE
Remove Bacteria authorship

### ISSUE THREAD

**Issue by mdoering** [CatalogueOfLife/data#223]
`Bacteria Cavalier-Smith, 2002` is the only kingdom with authorship. Better remove it via a decision?

https://www.catalogueoflife.org/data/taxon/B

**Comment by yroskov** [comment]
No, all other uninomials will get authorship - gradually, through updates.

**Comment by mdoering** [comment]
So what is the authorship for Animalia Plantae and other kingdoms and phyla managed by COL?
Then we need to change those

**Comment by yroskov** [comment]
> what is the authorship for Animalia Plantae and other kingdoms and phyla managed by COL

This is a question to the Hierarchy working group in Taxonomy Group

**Comment by mdoering** [comment]
Actually `Bacteria Cavalier-Smith, 2002` as the name authorship is wrong.
Cavalier-Smith did not publish the name _Bacteria_ for the first in 2002. It has been is use for a long time and I am not sure if ICN actually deals with authorship for higher ranks. ICZN surely does not.

Cavalier-Smith really is the concept reference. It should be [Bacteria sensu Cavalier-Smith, 2002](https://en.wikipedia.org/wiki/Cavalier-Smith%27s_system_of_classification) which should go into accordingTo.
Maybe that is sth for ITIS to change?

**Comment by yroskov** [comment]
Indeed, what should be present in authorstrings of high taxa is very tricky thing. It's a subject for good research.

**Comment by DaveNicolson** [comment]
Technically, at least under the "zoological" Code, there is no "priority" above the rank of superfamily, so there is no requirement to use any particular authorship, although there are conventions. I find authorship at such ranks to be troublesomely unstable, so I lean away from including them unless there is a well-accepted source following well-accepted authorship choices. But when a provider includes them we allow it, cringing a little bit.

Is there a particular reason for either excluding all or requiring all in COL?

**Comment by mdoering** [comment]
No. But we differ between taxonomic references and nomenclatural authorships.
For most higher taxa I think it is better to use `accordingTo=X` which gets rendered as `sensu X` in a names "authorship".
It makes clear we refer to where the taxonomic concept was established, not the name.

**Comment by DaveNicolson** [comment]
I guess no matter what you do some will be incorrect. There are applicable rules for names above superfamily in the zoological Code, just not including Priority. So the name must be published in a way that makes it available. But I have no way of knowing which names have authorship best considered "sensu" and which are for nomenclatural availability. I have my doubts as to whether nomenclatural availability is even considered by some authors for names above superfamily, but others seem very focused on it. In any case, I am not sure of the best approach to this and defer to you, having commented already.

**Comment by gdower** [comment]
One possible solution might be to not display uninomial authorships in the classification tree. If people want to view the uninomial authorships, the uninomial authorships would still be displayed the name page and in the API. Eventually when we show statistics in the tree, having the uninomial authorships might make the name strings too lengthy anyway.

**Comment by yroskov** [comment]
I like this idea very much: authorstrings with uninomials should not be displayed in the Tree in final product at CoL portal.

However, I need authorstrings with uninomials in the Clearinghouse Tree as they are now.

**Comment by mdoering** [comment]
I don't mind hiding authorship for uninomials in the tree. Or maybe just for suprageneric or suprafamily names. If you like this let's discuss this in a portal issue.

But it does not solve the issue of a wrong authorship. We should try to use taxonomic and nomenclatural authorship correctly like we try to avoid chresonym authorships. If it is impossible for ITIS to know what the authorship refers to we should apply a decision to fix this (not sure if we can actually apply accordingTo now, this might need a code change).

@DaveNicolson it seems indeed [ITIS has a mix of both authorship types](https://data.catalogueoflife.org/dataset/2144/names?facet=rank&facet=issue&facet=status&facet=nomStatus&facet=nameType&facet=field&field=combination%20authors&limit=50&offset=0&rank=infrakingdom&rank=infraphylum&rank=superphylum&rank=kingdom&rank=section&rank=superclass&rank=subkingdom&rank=infraclass&rank=subphylum&rank=superorder&rank=infraorder&rank=phylum&rank=subclass&rank=class&rank=suborder&rank=order&sortBy=taxonomic). There are even [basionym authors given for 2 taxa](https://data.catalogueoflife.org/dataset/2144/names?facet=rank&facet=issue&facet=status&facet=nomStatus&facet=nameType&facet=field&field=basionym%20authors&limit=50&offset=0&rank=infrakingdom&rank=infraphylum&rank=superphylum&rank=kingdom&rank=section&rank=superclass&rank=subkingdom&rank=infraclass&rank=subphylum&rank=superorder&rank=infraorder&rank=phylum&rank=subclass&rank=class&rank=suborder&rank=order&sortBy=taxonomic).

From the authorship years it seems [all of Bacteria is citing taxon concepts](https://data.catalogueoflife.org/dataset/2144/names?TAXON_ID=50&facet=rank&facet=issue&facet=status&facet=nomStatus&facet=nameType&facet=field&field=combination%20authors&limit=50&offset=0&rank=infrakingdom&rank=infraphylum&rank=superphylum&rank=kingdom&rank=section&rank=superclass&rank=subkingdom&rank=infraclass&rank=subphylum&rank=superorder&rank=infraorder&rank=phylum&rank=subclass&rank=class&rank=suborder&rank=order&sortBy=taxonomic). But [Animalia mostly has names from pre 1900](https://data.catalogueoflife.org/dataset/2144/names?TAXON_ID=202423&facet=rank&facet=issue&facet=status&facet=nomStatus&facet=nameType&facet=field&field=combination%20authors&limit=50&offset=0&rank=infrakingdom&rank=infraphylum&rank=superphylum&rank=kingdom&rank=section&rank=superclass&rank=subkingdom&rank=infraclass&rank=subphylum&rank=superorder&rank=infraorder&rank=phylum&rank=subclass&rank=class&rank=suborder&rank=order&sortBy=taxonomic) which are likely nomenclatural references.

**Comment by mdoering** [comment]
There are 2 ways in ColDP you can indicate the taxon concept authorship:

a) the proper way is to create a reference record for it and cite it via [accordingToID](https://github.com/CatalogueOfLife/coldp/blob/master/README.md#accordingtoid) in the Taxon/Synonym/NameUsage table.

b) use _sensu_ in front of the name authorship, e.g. `Bacteria sensu Cavalier-Smith, 2002` . This tells the name parser it is the taxonomic reference and it will be moved to accordingTo during imports.

**Comment by DaveNicolson** [comment]
I seriously doubt that most authors have even thought about authorship at those levels (much less properly researched them for original nomenclatural availability), and I cannot imagine it would be anything but a **_huge_** diversion of time to focus on these, although I recognize you feel the need to do something here. When the specialists give us names at these ranks with authorship, we follow them unless there is a reason to omit them, and in those cases I would be very happy to omit them. I think this is one of the reasons why ITIS was designed to only *require* authorship for names at the rank of genus and below, but to *allow* them at the higher levels.

I think the proposal to omit them in COL makes good sense. I don't know how to advise you for the clearinghouse. My assumption is that MOST uses at higher level are "sensu" style, but probably not all of them, and I have no quick fix for telling which is which without potentially a lot of research.

It's a shame there is no "lowest common denominator" for authorship that could be one type vs. the other without needing to know which was which.

**Comment by mdoering** [comment]
Agree that removal of authorships for any names above family would be best.
If we are unclear what the "authorship" refers to I think it is best to not have it or to have a mix of different meanings.
As the codes cover family group names, I would propose to keep authorships for family and genera as a new thing in COL.

**Comment by TonyRees** [comment]
Here is a copy of remarks on this subject that I posted to the CoL "Discussion points for Taxonomy Group" document on Google docs, copied here in case they would otherwise be lost, also so they appear in more context:

1: I support using authorships [note: also including these in CoL display] for names above family. For interest, at kingdom level one source (the Taxonomicon, Netherlands) currently has:
- Kingdom Animalia Linnaeus, 1758
- Kingdom Plantae Haeckel (1866)
- Kingdom Fungi R.T. Moore (1980)
- Kingdom Protozoa Goldfuss, 1818
- Kingdom Chromista Cavalier-Smith (1981)
- Kingdom Bacteria Cavalier-Smith 2002 see Domain Bacteria Woese et al. 1990 (**NB discussed further below)
- Kingdom Archaea see Domain Archaea Woese et al. 1990

The use of parentheses seems a bit inconsistent. In any case, these could be a starting point for further checks.

2: In the longer term, I think CoL probably should manage its own classification down to family, but in the short term, it probably has no choice but to accept authorships for these as supplied by "trusted partners" - maybe ITIS could be the default (if it wishes), then others where not in ITIS??

3: "Bacteria" is a slightly curly case (especially when it has been used twice for genera as well: Bacteria Berthold in Latreille, 1827 (an accepted name in Phasmatodea) and Bacteria Agassiz, 1846 (an unaccepted emendation of Bactiria Fischer 1808, the latter apparently a nomen nudum)). In Cavalier-Smith, 1998 "A revised six-kingdom system of life", kingdom Bacteria is given as Kingdom Bacteria Cohn 1870 stat. nov. Cavalier-Smith 1983. So (to me) this establishes the name at that taxonomic rank, but not necessarily the concept (C-S included Archaea as well, which CoL now does not). The nearest to the concept would be from Woese, Magrum & Fox, 1978, who proposed "urkingdom Archaebacteria" alongside "urkingdom Bacteria". Thus if you accept this as equivalent to kingdom, the name for the current concept would be "Kingdom Bacteria Cohn, 1870, stat. nov. Cavalier-Smith, 1983, sensu Woese et al., 1978" (or similar...) (subsequently, Woese, Kandler & Wheelis, 1990 elevated these to domains, above kingdom...) -  or there may be another citation form/original proposal that I have missed...
